osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -3244,6 +3244,36 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.svg"
       }
-    }
+    },
+	{
+      "description": "MUSE Governance Token",
+      "type_asset": "cw20",
+      "address": "juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3",
+      "denom_units": [
+        {
+          "denom": "IBC/6B982170CE024689E8DD0E7555B129B488005130D4EDA426733D552D10B36D8F",
```

Comment:
please make the letter: 'IBC' lower case. -> "ibc/6B9...

Reply:
updated